tests/bdd/features/helm_chart/install.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Install Mayastor
+  Install Mayastor on a kubernetes cluster, using helm
+
+  Background:
+    Given: A Kubernetes cluster
+
+  Scenario: Mayastor helm chart install
+    Given worker nodes on the kubernetes cluster are labelled with the label 'openebs.io/engine=mayastor'
+    And a kubernetes namespace to install the Mayastor helm chart
+    When the Mayastor helm chart is installed with callhome disabled and at most 3 replicas of etcd
```

Comment:
Should callhome be enabled but with sendReport false?

Reply:
Resolving this conversation, because this test is no longer a part of this PR.